ui/apos/components/AposExportModal.vue

```diff
@@ -452,15 +433,18 @@ export default {
   width: 100%;
 }
 
-.apos-export__section-container {
-  height: var(--container-height);
-  transition: height 200ms linear;
-}
+.apos-export__section--related-documents {
+  overflow: hidden;
+  height: 315px;
 
-.slide-enter-from, .slide-leave-to {
-  .apos-export__section-container {
+  &.slide-enter-active,
+  &.slide-leave-active {
+    transition: all 200ms linear;
+  }
+
+  &.slide-enter-from,
+  &.slide-leave-to {
     height: 0;
   }
```

Comment:
we now use the provided `<translation>` component, and stop mixing it with our own homemade transitions

Reply:
you mean `<transition>` ?